Dockerfile

```diff
@@ -13,4 +13,5 @@ USER app
 
 COPY . /usr/src/app
 
+ENV ENGINE_CONFIG="/config.json"
```

Comment:
This path could just be hard-coded [here](https://github.com/ivantsepp/codeclimate-scss-lint/blob/master/bin/codeclimate-scss-lint#L8) since it will always be bind-mounted. That might be a bit clearer